os/binfmt/binfmt_loadbinary.c

```diff
@@ -203,6 +184,22 @@ int load_binary(int binary_idx, FAR const char *filename, load_attr_t *load_attr
 #ifdef CONFIG_SAVE_BIN_SECTION_ADDR
 		elf_save_bin_section_addr(bin);
 #endif
+
+#if (defined(CONFIG_ARMV7M_MPU) || defined(CONFIG_ARMV8M_MPU))
```

Comment:
Let's split the commit. Could you add why this should be moved in description?

Reply:
Done.